ui/uihandler.go

```diff
@@ -183,6 +185,9 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 		log.Fatalf("expected http.ResponseWriter to be an http.Flusher")
 	}
 	out := io.Writer(os.Stderr)
+	if len(percList) == 0 {
```

Comment:
can you make the difference between p= isn't provided, in which case we use defaults and p= is empty in which case we should respect the will of the caller to not calculate any % ?